### PR TITLE
Turn exercise 032 to practice

### PR DESCRIPTION
**Pull request by Ozyugoo**

### CODE REVIEW

**Comment by Ozyugoo** [comment]
@cuihtlauac, kindly check it, Thank you.

File: practice/032/ex.ml [hunk 1]
```diff
@@ -0,0 +1,15 @@
+open OUnit2
+
+module type Testable = sig
+  val gcd : int -> int -> int
+end
+
+module Make(Tested: Testable) : sig val v : test end = struct
+  let v = "gcd" >::: [
+    "nil" >:: (fun _ -> assert_equal 13 (Tested.gcd 13 0));
+    "cons" >:: (fun _ -> assert_equal 2 (Tested.gcd 20536 7826));
```
Comment (cuihtlauac) [inline practice/032/ex.ml]: Change the name of the tests into `gcd_13_0` and `gcd_20536_7826`

**Review by cuihtlauac — CHANGES_REQUESTED** [review]

File: practice/032/ex.ml (hunk 1 above)
Comment (Ozyugoo) [reply practice/032/ex.ml]: Could you take a look at it, is this what ask me to do?

**Review by cuihtlauac — APPROVED** [review]
Thnaks @Ozyugoo

**Pull request merged**